### ISSUE TITLE
Sapper viability on Netlify

### ISSUE THREAD

**Issue by n-edwards** [sw-yx/ssg#8]
Hi @sw-yx. Thanks for your efforts trying to get Sapper working better with the Netlify platform. Would you say it's viable to use Sapper for a small to medium-sized site or PWA currently? Would something like your [Gatsby JAMstack app](https://github.com/sw-yx/jamstack-hackathon-starter/) be doable with Sapper, or would the issues you've [detailed](https://www.npmjs.com/package/@ssgjs/sapper) be major hurdles? Would your SSG be needed?
In SSG's readme, you also indicated "Netlify doesn't support datastores"? Any further details on that?

Would you recommend against trying to build something non-trivial with only Svelte, no Sapper?

---

I'm really liking Svelte's "less code" philosophy, and would love to get something deployed with Sapper, without having to switch to a generator that takes a React, Vue, or Angular approach.

Sapper seems to have been targeted at SSR, but after working through some of the hurdles with that, using [Amplify](https://stackoverflow.com/questions/58383122/how-to-use-aws-amplify-with-sapper/58518977#58518977) and [Firebase](https://dev.to/eckhardtd/how-to-host-a-sapper-js-ssr-app-on-firebase-hmb) tools, I'm came across Netlify's dynamic JAMstack [pitch](https://www.netlify.com/blog/2017/06/06/jamstack-vs-isomorphic-server-side-rendering/) and "one workflow" philosophy, which seemed like a better setup.

I think Sapper team is getting the message around a desire for static deploys + serverless functions over SSR? Have you seen this [thread](https://github.com/sveltejs/sapper/issues/564#issuecomment-529663125) and fork?

On the Netlify side, I'd love to see a Sapper [configuration](https://docs.netlify.com/configure-builds/common-configurations) soon, or even just a [template](https://templates.netlify.com/).
I'm seeing that the Netlify CLI does detect Sapper projects, so I'm hoping to see the pairing gain traction. I'd be happy to sound off in Netlify Community or in Sapper's issues if you think that could help.

**Comment by swyxio** [comment]
yea everything you can do with gatsby you can do with sapper/netlify, it might just take a bit more elbow grease cos sapper is newer. 

ive seen that thread but its more about serving sapper from serverless functions which we dont really promote at netlify (you can technically do it it will just cost you).

i’m still not done with ssg, which is why i havent bothered to do much marketing yet. but yea you can see it in action on my site and the svelte community site.

**Comment by n-edwards** [comment]
Awesome, thanks. I also came across [Pinafore](https://nolanlawson.com/2018/04/09/introducing-pinafore-for-mastodon/), which seems to be a full app working with Sapper.

Ah, OK, it did seem like the Now v2 thread was up to something a bit more complex. That sounds along the lines of what I was wrestling with in Firebase.

Looking forward to your updates! A visual editor in particular sounds nuts. I didn't even know Sapper Studio was a thing until I saw your roadmap.
